ci/scripts/go-executable-build.sh

```
@@ -0,0 +1,33 @@
+do
+	platform_split=(${platform//\// })
+	GOOS=${platform_split[0]}
+	GOARCH=${platform_split[1]}
+	output_name=$package_name
+	if [ $GOOS = "windows" ]; then
+		output_name+='.exe'
+	fi
+	echo "building $package_name for $GOOS/$GOARCH..."
+	mkdir -p $output_path/$GOOS/$GOARCH
+	env GOOS=$GOOS GOARCH=$GOARCH go build -o $output_path/$GOOS/$GOARCH/$output_name $3
+	if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)